src/main/java/sparksoniq/semantics/types/SequenceType.java

```diff
@@ -22,6 +22,8 @@
 
```

Comment:
SequenceType.ToString() is used in multiple places for serialization of the SequenceType. I think this method should be implemented to properly resemble (or match) the serialization of the removed class FlworVarSequenceType.

Reply:
I added toString(), with a serialization closer to JSONiq.